Masquerade4997Library/src/main/java/Library4997/MasqSensors/MasqVuforia.java

```diff
@@ -190,12 +190,12 @@ private String position(String target){
     public String[] getDash() {
         return new String[]{
                 getName() +
-                "TargetOneSeen" + Boolean.toString(isSeen(targetOne)),
+                        "TargetOneSeen" + Boolean.toString(isSeen(targetOne)),
```

Comment:
Why?

Reply:
I didn’t change this maybe it was an accident